packages/xo-server-backup-reports/src/index.js

```diff
@@ -66,8 +66,11 @@ const logError = e => {
   console.error('backup report error:', e)
 }
 
-const UNHEALTHY_VDI_CHAIN_ERROR = 'unhealthy VDI chain'
 const NO_SUCH_OBJECT_ERROR = 'no such object'
+const UNHEALTHY_VDI_CHAIN_ERROR = 'unhealthy VDI chain'
+const UNHEALTHY_VDI_CHAIN_LINK =
```

Comment:
Why do you need this variable?

Reply:
It uses to redirecte the used to the dedicated documentation.

Reply:
Used only once in the `const` right below: inline.